src/13-core-functions.md

```diff
@@ -122,6 +122,6 @@ io.println(hello[5]);
 
 Output:
 ```
-h
+!
 (nil)
```

Comment:
umm... `(nil)` should have been `!` and `h` should be as it is 😛

Reply:
:facepalm: